controllers/storagecluster/cephcluster.go

```diff
@@ -492,6 +496,21 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 		MaxLogSize:  &maxLogSize,
 	}
 
+	strVal := util.MustGetEnv(desiredCephxKeyGenEnvVarName)
+	intVal, err := strconv.Atoi(strVal)
```

Comment:
These vars are in function scope so we need to give them proper names
```suggestion
	desiredCephxKeyGenAsString := util.MustGetEnv(desiredCephxKeyGenEnvVarName)
	desiredCephxKeyGen, err := strconv.Atoi(strVal)
```